explain-overview.md

```diff
@@ -6,500 +6,52 @@ aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/und
-4 rows in set (0.00 sec)
-```
+  In this case, the `build` operator finds the internal `RowID` for rows that match in the index `idx_a`. The `probe` operator then retrieves these rows from the table.
 
-After adding the index, use `IndexScan_24` to directly read the data that meets the `start_date BETWEEN '2017-07-01 00:00:00' AND '2017-07-01 23:59:59'` condition. The estimated number of rows to be scanned decreases from 19117643.00 to 8166.73. In the test environment, the execution time of this query decreases from 50.41 seconds to 0.01 seconds.
+* `estRows` shows an estimate of the number of rows TiDB expects to process. This number might be based on dictionary information, such as if the access method is based on a primary or unique key, or it could be based on statistics such as a CMSketch or histogram.
 
-## Operator-related system variables
+* `task` shows where an operator is performing the work. A task of `cop[tikv]` indicates that work is being completed inside TiKV as part of the coprocessor. TiDB will attempt to push as much of the query to the data as possible, since this reduces the amount of data sent across the network. `root` tasks indicate that the operator is executed inside TiDB.
 
-Based on MySQL, TiDB defines some special system variables and syntax to optimize performance. Some system variables are related to specific operators, such as the concurrency of the operator, the upper limit of the operator memory, and whether to use partitioned tables. These can be controlled by system variables, thereby affecting the efficiency of each operator.
+* `access object` shows the table, partition and index that is being accessed. The parts of the index will also be shown, as in the case above the column `a` from the index was used. This can be useful in cases where you have composite indexes.
 
-## See also
+* `operator info` shows additional details about the access, such as which conditions were able to be pushed down:
 
-* [EXPLAIN](/sql-statements/sql-statement-explain.md)
-* [EXPLAIN ANALYZE](/sql-statements/sql-statement-explain-analyze.md)
-* [ANALYZE TABLE](/sql-statements/sql-statement-analyze-table.md)
-* [TRACE](/sql-statements/sql-statement-trace.md)
-* [System Variables](/system-variables.md)
+    * `range: [1,1]` shows that the predicate from the where clause of the query (`a = 1`) was pushed right down to TiKV (the task is of `cop[tikv]`).
+    * `keep order:false` shows that the semantics of this query did not require TiKV to return the results in order. If the query were to be modified to require an order (such as `SELECT * FROM t WHERE a = 1 ORDER BY id`), then this would show `keep order:true`.
+    * `stats:pseudo` shows that the estimates shown in `estRows` were based on a heuristic and might not be accurate. TiDB will periodically update statistics as part of a background operation. A manual update can also be performed by running `ANALYZE TABLE t`.
```

Comment:
It seems that "heuristic" is more commonly used as a verb. Is there an omitted word in "based on a heuristic and"? Or can we just remove that part?
```suggestion
    * `stats:pseudo` shows that the estimates shown in `estRows` might not be accurate. TiDB periodically updates statistics as part of a background operation. A manual update can also be performed by running `ANALYZE TABLE t`.
```